Jenkinsfile

```diff
@@ -122,7 +122,7 @@ pipeline {
                                     branch: env.BaseBranch,
                                     withSubmodules: true
                         functionalTest inst_repos: daosRepos(),
-                                       inst_rpms: functionalPackages(1, next_version),
+                                       inst_rpms: functionalPackages(1, next_version, "tests-internal"),
```

Comment:
Is this just a transitional step to baking this new test name right into `pipeline-lib`?

Reply:
Yes.